### ISSUE TITLE
linux or osx or xp support

### ISSUE THREAD

**Issue by vchong** [league1991/CodeAtlasSublime#1]
Hi,

The instructions only seem to support Windows installation. Do you have support for Linux or OS X?

Thanks!

**Comment by vchong** [comment]
Windows XP doesn't work either. :(

**Comment by vchong** [comment]
After clicking 'Start Atlas', nothing happens on os x (maybe expected since there's no understand.pyd file on os x version of Understand). On xp, see a Dos window appear and disappear quickly, but nothing else after that.

**Comment by league1991** [comment]
@vchong You are right. Linux and mac aren't supported because understand doesn't has such versions.

For xp, you can try to run main.py manually. If you succeed, then check codeView.bat. The overall invocation chain is sublime -> codeView.bat -> main.py. I think there is something wrong in this chain.

**Comment by vchong** [comment]
@league1991 Thanks very much for your support!

> Linux and mac aren't supported because understand doesn't has such versions.

? There are Linux and mac versions of understand tool. https://scitools.com/download/archive

I replaced xp with win7 for now. Will try your suggestion if use xp again. Thanks!

**Comment by vchong** [comment]
> For xp, you can try to run main.py manually.

So clicking on 'Start Atlas' didn't work, but running main.py manually works! Thanks!

Since there are Linux and mac versions of understand tool, do you think you'll support Linux and mac in the future?

**Comment by league1991** [comment]
@vchong Oh, I didn't find that link before.
Theoretically, linux or mac can be supported if understand also provides python API in these platforms.
The difficulty is that I have to find a pyqt whose python version is compatible with understand's python API.
I will try to do that if I have time and tell you when done.
